### PR TITLE
Added Bootstrap

### PR DESCRIPTION
**Pull request by Hunter2065**
I added Bootstrap to GemFile and also created folders for it with moving some files by Adriy's tutorial

### CODE REVIEW

**Review by montashevych — APPROVED** [review]

File: app/frontend/js/bootstrap_js_files.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// import 'bootstrap/js/src/alert'
+// import 'bootstrap/js/src/button'
+// import 'bootstrap/js/src/carousel'
+//import 'bootstrap/js/src/collapse'
```
Comment (andrjuha01) [inline app/frontend/js/bootstrap_js_files.js:4]: Fix indentation

**Review by andrjuha01 — COMMENTED** [review]
And add a screenshot that bootstrap is used on local machine

File: app/frontend/js/bootstrap_js_files.js (hunk 1 above)
Comment (Hunter2065) [reply app/frontend/js/bootstrap_js_files.js:4]: fixed

**Comment by Hunter2065** [comment]
> And add a screenshot that bootstrap is used on local machine

image

**Review by andrjuha01 — APPROVED** [review]

**Pull request merged**